signal_scanner_bot/env.py

```diff
@@ -31,7 +30,8 @@ def _env(
     fail: bool = True,
     default: Any = None,
 ) -> Any:
-    """
+    """Get environment variable.
```

Comment:
Are these really meant to be on the same line as the opening `"""`?

Reply:
That's the way it's "supposed" to be (and how we do for work) but honestly, I hate it! 😄 I'm gonna change this rule lol